src/lib/holocene/custom-button.svelte

```diff
@@ -0,0 +1,78 @@
+<script lang="ts">
```

Comment:
What does `<CustomButton />` do/have that `<Button />` doesn't?

Reply:
Yeah good call out. It's basically a button with much more flexibility on styles with a few base styles. There were a lot of little differences in the designs and I didn't want to keep adding to existing, but let me see if they are needed. A lot of this was driven from the advanced filter designs that aren't being used yet.

Reply:
Moved this out of holocene and into advanced-filters director to not create confusion. I don't want to lose it but it's not used until we get advanced filters in.